File: src/main/java/com/rtomyj/yugiohAPI/helper/enumeration/browse/MonsterAssociationExpression.java
package com.rtomyj.yugiohAPI.helper.enumeration.browse;

public enum MonsterAssociationExpression
{

    LEVEL_EXPRESSION("\"level\": \"%s\"")
    , RANK_EXPRESSION("\"rank\": \"%s\"")
    , LINK_RATING_EXPRESSION("\"linkRating\": \"%s\"");


    private final String expression;

    MonsterAssociationExpression(final String expression)
    {
        this.expression = expression;
    }


    @Override
    public String toString()
    {
        return this.expression;
    }

}

File: src/main/kotlin/com/rtomyj/skc/util/enumeration/MonsterAssociationExpression.kt
package com.rtomyj.skc.util.enumeration

enum class MonsterAssociationExpression(
    private val expression: String
) {
    LEVEL_EXPRESSION("\"level\": \"%s\""),
    RANK_EXPRESSION("\"rank\": \"%s\""),
    LINK_RATING_EXPRESSION("\"linkRating\": \"%s\"");


    override fun toString(): String = expression
}